### PR TITLE
Clean up GiniConfiguration and related files

### PR DESCRIPTION
**Pull request by vizaknaidavid**
PIA-3448

[PIA-3448]: https://ginis.atlassian.net/browse/PIA-3448?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: BankSDK/GiniBankSDK/Documentation/source/Customization guide.md [hunk 1]
```diff
@@ -40,14 +40,7 @@ Some background and text colors use the `GiniColor` type with which you can set
...
 
-##### 2. Notice
-<center><img src="img/Customization guide/Notice.jpg" height="70"/></center>
-- Information background color &#8594;  `GiniBankConfiguration.noticeInformationBackgroundColor`
-- Information text color &#8594;  `GiniBankConfiguration.noticeInformationTextColor`
-- Error background &#8594;  `GiniBankConfiguration.noticeErrorBackgroundColor`
-- Error text color `GiniBankConfiguration.noticeErrorTextColor`
-
-##### 3. Tooltip
+##### 2. Tooltip
```
Comment (zladzeyka) [inline BankSDK/GiniBankSDK/Documentation/source/Customization guide.md:43]: [Suggestion] Tooltip is not used anymore as well 😉

File: BankSDK/GiniBankSDK/Documentation/source/Customization guide.md [hunk 2]
```diff
@@ -40,14 +40,7 @@ Some background and text colors use the `GiniColor` type with which you can set
 - Item font &#8594;  `GiniBankConfiguration.navigationBarItemFont`
 - Title font &#8594;  `GiniBankConfiguration.navigationBarTitleFont`
 
-##### 2. Notice
```
Comment (zladzeyka) [inline BankSDK/GiniBankSDK/Documentation/source/Customization guide.md:43]: [Clarification] I was 🤔 thinking we use `Notice` when some import errors occur.

File: BankSDK/GiniBankSDK/Documentation/source/Customization guide.md [hunk 3]
```diff
@@ -138,7 +126,6 @@ Some background and text colors use the `GiniColor` type with which you can set
 - Rotation button image &#8594;  <span style="color:#009EDF">*reviewRotateButton*</span> image asset
```
Comment (zladzeyka) [inline BankSDK/GiniBankSDK/Documentation/source/Customization guide.md:126]: [Suggestion] Rotation functionality was removed.

File: BankSDK/GiniBankSDK/Documentation/source/Integration.md [hunk 4]
```diff
@@ -260,7 +260,6 @@ The components that can be found in the library are:
 * **Review**: Allows to check the quality of one or several images and the ability to delete when multipage is enabled (`ReviewViewController`).
 * **Analysis**: Provides a UI for the analysis process of the document by showing the user capture tips when an image is analyzed or the document information when it is a PDF. In both cases an image preview of the document analyzed will be shown (`AnalysisViewController`).
 * **Help**: Helpful tutorials indicating how to use the open with feature, which are the supported file types and how to capture better photos for a good analysis (`HelpMenuViewController`).
-* **No results**: Shows some suggestions to capture better photos when there are no results after an analysis (`ImageAnalysisNoResultsViewController`).
```
Comment (zladzeyka) [inline BankSDK/GiniBankSDK/Documentation/source/Integration.md:263]: [Clarification] Why did we remove it? We still have `No results` screen 😉

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/ReturnAssistantConfiguration.swift [hunk 5]
```diff
@@ -22,53 +22,6 @@ public final class ReturnAssistantConfiguration: NSObject {
      - returns: Instance of `ReturnAssistantConfiguration`.
      */
     public override init() {}
-    // MARK: Return Assistant
```
Comment (zladzeyka) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/ReturnAssistantConfiguration.swift:25]: We need to revert the changes in [ReturnAssistantConfiguration] 😊

File: CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md [hunk 6]
```diff
@@ -63,14 +63,7 @@ Images customization is done via overriding of [GiniImages.xcassets](https://git
...
 
-##### 2. Notice
-<center><img src="img/Customization guide/Notice.jpg" height="70"/></center>
-- Information background color &#8594;  `GiniConfiguration.noticeInformationBackgroundColor`
-- Information text color &#8594;  `GiniConfiguration.noticeInformationTextColor`
-- Error background &#8594;  `GiniConfiguration.noticeErrorBackgroundColor`
-- Error text color `GiniConfiguration.noticeErrorTextColor`
-
-##### 3. Tooltip
+##### 2. Tooltip
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md:66]: [Suggestion] Tooltips are not used anymore.

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera2/Camera2ViewController.swift [hunk 7]
```diff
@@ -109,7 +109,6 @@ public final class Camera2ViewController: UIViewController, CameraScreen {
 
     private func setupView() {
         edgesForExtendedLayout = []
-        view.backgroundColor = giniConfiguration.cameraContainerViewBackgroundColor.uiColor()
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera2/Camera2ViewController.swift:112]: [Clarification] If we remove this Configuration we need to use a color from `GiniColors` palette instead.

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/CameraPreviewViewController.swift [hunk 8]
```diff
@@ -122,7 +122,6 @@ final class CameraPreviewViewController: UIViewController {
                                                name: NSNotification.Name.AVCaptureDeviceSubjectAreaDidChange,
                                                object: camera.videoDeviceInput?.device)
         view.backgroundColor = GiniColor(light: UIColor.GiniCapture.dark1, dark: UIColor.GiniCapture.dark1).uiColor()
-        previewView.drawGuides(withColor: giniConfiguration.cameraPreviewCornerGuidesColor)
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/CameraPreviewViewController.swift:125]: [Clarification] If we remove this Configuration we need to use a color from the `GiniColors` palette instead.

File: HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/ComponentAPICoordinator.swift [hunk 9]
```diff
@@ -356,8 +356,7 @@ extension ComponentAPICoordinator: UINavigationControllerDelegate {
         
         if toVC is CameraViewController &&
             (fromVC is ReviewViewController ||
```
Comment (zladzeyka) [inline HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/ComponentAPICoordinator.swift:358]: [Suggestion] please revert health-related changes 😉

File: CaptureSDK/GiniCaptureSDK/Tests/GiniCaptureSDKTests/ImageNoResultsViewControllerTests.swift [hunk 10]
```diff
@@ -1,76 +0,0 @@
-//
-//  ImageAnalysisNoResultsViewControllerTests.swift
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Tests/GiniCaptureSDKTests/ImageNoResultsViewControllerTests.swift:2]: [Clarification] Do we have `NoResultsViewControllerTests`?

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Document picker/DocumentPickerCoordinator.swift [hunk 11]
```diff
@@ -182,19 +182,6 @@ public final class DocumentPickerCoordinator: NSObject {
 
         documentPicker.allowsMultipleSelection = giniConfiguration.multipageEnabled
 
-        if let tintColor = giniConfiguration.documentPickerNavigationBarTintColor {
-            // Starting with iOS 11.0, the UIDocumentPickerViewController navigation bar almost can't be customized,
-            // only being possible to customize the tint color. To avoid issues with custom UIAppearance styles,
-            // this is reset to default, saving the current state in order to restore it during dismissal.
-            saveCurrentNavBarAppearance()
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Document picker/DocumentPickerCoordinator.swift:189]: [Clarification] Are those changes not needed for iOS 12?

**Review by zladzeyka — CHANGES_REQUESTED** [review]
I'm a little bit surprised 😲 the ticket was about cleanup colors from `No Results` 🙈 and I see a lot of changes 🥹 inside the core - it can cause a huge regression 🥲 we shouldn't so many changes for a small task 😇.

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/ReturnAssistantConfiguration.swift (hunk 5 above)
Comment (vizaknaidavid) [reply BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/ReturnAssistantConfiguration.swift:25]: We can revert it, but these are not used anywhere in the app, other parts of the ReturnAssistantConfiguration is untouched which is used.

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera2/Camera2ViewController.swift (hunk 7 above)
Comment (vizaknaidavid) [reply CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera2/Camera2ViewController.swift:112]: This backgroundColor is not visible anywhere because of the CameraPreviewVC is above this.

File: BankSDK/GiniBankSDK/Documentation/source/Integration.md (hunk 4 above)
Comment (vizaknaidavid) [reply BankSDK/GiniBankSDK/Documentation/source/Integration.md:263]: Here the old code was referring to a VC that is not used anymore: ImageAnalysisNoResultsViewController. The file was also deleted as part of this MR.

**Review by zladzeyka — APPROVED** [review]
👏🏻😉
